### PR TITLE
Edited some text

### PR DESCRIPTION
**Pull request by svmorris**
Not very polished, but made a few changes to the text.

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -23,39 +23,50 @@
...
 		<div class="bg-flipped grid-left">
+			<div>
+				<h1>A dynamic source code analysis tool...</h1>
+			</div>
+		</div>
+		<div class="bg grid-left">
 			<img src="/images/screenshots/1.png" alt="Screenshot 1" />
 			<div>
-				<h1>Collect logs from multiple remote sources concurrently</h1>
+				<h1>That collects logs in one friendly interface.</h1>
```
Comment (STBoyden) [inline src/routes/index.svelte]: Can you remove the full stop?

Comment (svmorris) [reply src/routes/index.svelte]: just remove it mate

Comment (STBoyden) [reply src/routes/index.svelte]: No

**Comment by STBoyden** [comment]
I love your commit messages! They seem really high quality! This is not sarcasm!

**Comment by STBoyden** [comment]
~Fuck you~

**Comment by kozmer** [comment]
> I love your commit messages! They seem really high quality! This is not sarcasm!

Please keep the comments relevant. Thanks.

**Comment by svmorris** [comment]
> Fuck you
Please stay respectful @STBoyden 
https://tenor.com/view/davy-enardo-discord-gif-19344068https://tenor.com/view/davy-enardo-discord-gif-19344068

**Comment by kozmer** [comment]
> > Fuck you
> > Please stay respectful @STBoyden
> > https://tenor.com/view/davy-enardo-discord-gif-19344068https://tenor.com/view/davy-enardo-discord-gif-19344068

[stay respectful](https://i.pinimg.com/736x/6d/49/19/6d4919580355cfdf5b1db6576d217fe2.jpg)

File: src/routes/index.svelte [hunk 2]
```diff
@@ -23,39 +23,53 @@
...
+					<li>The call stack</li>
+					<li>The log message</li>
+					<li>Where it was called</li>
+					<li>Some code context for the log</li>
+					<!--
+                        TODO: Make the screenshot more representative of these points.
+
+                        For example it would be better to have the image clearly show that the log (something simple, preferable 1 line)
+                        was called in a context of code. Maybe show the entire function with it.
+                    -->
```
Comment (STBoyden) [inline src/routes/index.svelte:58]: >:(

**Review by STBoyden — CHANGES_REQUESTED** [review]

File: src/routes/index.svelte (hunk 2 above)
Comment (STBoyden) [reply src/routes/index.svelte:58]: Too indented.

Comment (svmorris) [reply src/routes/index.svelte:58]: Can you please provide some more information on this.

> ":("

Is not very helpful unfortunately :slightly_frowning_face:

**Comment by svmorris** [comment]
[Stay toxic](https://cdn.discordapp.com/attachments/836564574294638605/1003439409263300678/stay-toxic.mp4)

**Pull request merged**